docker/docker-compose.dev.yml

```diff
@@ -21,6 +21,10 @@ services:
       - .env
     environment:
       - NODE_ENV=development
+    ulimits:
```

Comment:
This was necessary in my environment, or it wouldn't start

Reply:
What is your environment?

Reply:
This was podman on MacOS

Reply:
I'm fine if you want to put these dev changes in a separate PR and others can weigh in on them.

Reply:
Will do

Reply:
I have removed those changes from this PR, will open a separate one